packages/driver-interface/botRoutes.js

```diff
@@ -73,41 +82,42 @@ const init = (bot) => {
   })
 
   bot.command('/login', (ctx) => {
-    ctx.reply(`Skriv in ditt transport id`)
+    ctx.reply('Ange ditt transport-id')
   })
 
   bot.on('message', (ctx) => {
     const msg = ctx.message
 
-    if (msg.text) botServices.onLogin(msg.text, ctx)
+    /** Login attempt from command /login. */
+    if (msg.text.includes('pmv-')) {
```

Comment:
This if clause could include msg.text && msg.text.includes('pmv-')
since otherwise it crashed when the user sends location

Reply:
Good catch, will fix! There are probably more left overs from earlier functionality that is not relevant at this stage.

Reply:
Updated